### DISCUSSION TITLE
github-no-newline-at-a-end-of-file/

### DISCUSSION THREAD

**Discussion by giscus** [Hyeon9mak/Hyeon9mak.github.io#178] [Blog comments]
# github-no-newline-at-a-end-of-file/

GitHub 에 Pull Request 를 올리면 가끔 아래와 같은 금지 마크를 발견할 때가 있다.  

https://hyeon9mak.github.io/github-no-newline-at-a-end-of-file/

**Comment by ivvve** [comment]
좋은 정보가 담긴 글 잘 읽었습니다!

코딩도 중요하지만 이런 근본적인 것을 아는 개발자와 모르는 개발자는 차이가 있기 마련이더라구요 👍

**Comment by relkimm** [comment]
오! 한번씩 궁금했던 포인트인데 덕분에 오늘로써 제대로 알게 되었네요. 감사합니다 :)
